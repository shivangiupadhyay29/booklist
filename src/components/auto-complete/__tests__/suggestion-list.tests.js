import React from 'react';
import SuggestionList from '../suggestion-list.js';
import { shallow } from 'enzyme';

describe('Suggestion list component', () => {
  it('list component', () => {
    const mockFnHandler = jest.fn();
    const mockList = [
      {
        id: 0,
        title: 'book title',
        author: 'author name',
        summary: 'summary of the book text'
      }
    ];
    const wrapper = shallow(
      <SuggestionList list={mockList} addSelectedBook={mockFnHandler} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
