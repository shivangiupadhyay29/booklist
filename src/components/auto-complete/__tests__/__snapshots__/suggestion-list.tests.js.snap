// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Suggestion list component list component 1`] = `ShallowWrapper {}`;
